docs/source/app-configuration.rst

```diff
@@ -0,0 +1,157 @@
+=========================  ===========
+
+.. note:: Duo Security's Admin API is limited to two (2) requests per-minute. Therefore, setting the ``--timeout`` flag to any value between 10 and 60 will be of no additional value. A recommended timeout of 80 seconds will guarantee four (4) requests happen per-execution.
+
+2. Enter the required authentication information
+````````````````````````````````````````````````
+
+The above command will result in a few prompts asking for the required authentication information needed to configure this StreamAlert App.
+
+.. note:: After the last required authentication value is entered, the values are sent to AWS SSM's `Parameter Store <https://aws.amazon.com/ec2/systems-manager/parameter-store/>`_ as a ``SecureString`` to be used as part of this App's config. Due to this requirement, please ensure you have the correct and valid AWS credentials loaded before continuing.
+
+Example Prompts for Duo Auth
+''''''''''''''''''''''''''''
+
+.. code-block:: bash
+
+  Please supply the API URL for your duosecurity instance. This should be in a format similar to 'api-abcdef12.duosecurity.com': api-abcdef12.duosecurity.com
+
+  Please supply the secret key for your duosecurity Admin API. This should a string of 40 alphanumeric characters: 123424af2ae101d47d9704b783c940dffa825678
+
+  Please supply the integration key for your duosecurity Admin API. This should be in a format similar to 'DIABCDEFGHIJKLMN1234': DIABCDEFGHIJKLMN1234
+
+
+Once the above is completed, a logger statement similar to the following will confirm the configuration::
+
+  StreamAlertCLI [INFO]: App authentication info successfully saved to parameter store.
+  StreamAlertCLI [INFO]: Successfully added 'duo_prod_collector' app integration to 'conf/clusters/prod.json' for service 'duo_auth'.
+
+
+And the ``conf/clusters/prod.json`` file will be updated to include the configuration for this App:
+
+.. code-block:: json
+
+  {
+    "...": "...",
+    "modules": {
+      "...": "...",
+      "stream_alert_apps": {
+        "duo_prod_collector": {
+          "current_version": "$LATEST",
+          "interval": "rate(2 hours)",
+          "log_level": "info",
+          "memory": 128,
+          "timeout": 80,
+          "type": "duo_auth"
+        }
+      }
+    }
+  }
+
+
+The ``conf/sources.json`` file will also automatically update with the information the Rule Processor needs to accept input from this App:
+
+.. code-block:: json
+
+  {
+    "...": "...",
+    "stream_alert_app": {
+      "<prefix>_<cluster>_duo_auth_duo_prod_collector_app": {
+        "logs": [
+          "duo"
+        ]
+      }
+    }
+  }
+
+
+3. Deploy the new App and the Rule Processor
+````````````````````````````````````````````
+
+StreamAlert's Rule Processor must be aware of all input sources in order to process the data coming from them. As mentioned above, the ``conf/sources.json`` is automatically updated
```

Comment:
id nix this paragraph - its a detail that shouldn't be worried about if the CLI handles it